megalinter/descriptors/dart.megalinter-descriptor.yml

```diff
@@ -68,7 +68,8 @@ linters:
       ## stable dart sdk: https://dart.dev/get-dart#release-channels
       ## install alpine-pkg-glibc (glibc compatibility layer package for Alpine Linux)
       dockerfile:
-        - ARG DART_VERSION='2.8.4'
+        # renovate: datasource=dart
```

Comment:
There's two required fields in that regex.

Reply:
So what do I have to put `depName=dart`?

Reply:
I took this https://docs.renovatebot.com/presets-customManagers/#custommanagersdockerfileversions, and adapted it to allow changing our descriptor files too. Plus adapted our build scripts to keep the comment together with the arg when writing dockerfiles

More understandable docs: 
https://docs.renovatebot.com/modules/manager/regex/

Reply:
Renovate handles a lot of things, formats, sources, etc. But at some point, it needs to know where to look, and what to look for. In a custom manager, it can't know where to look, unlike if it was a npm package.json, it would look in the npm registry.

Reply:
@echoix as far as I understand the correct datasource is: https://docs.renovatebot.com/modules/datasource/dart-version/

And looking at this example we can use `depName=dart` as I said: https://github.com/renovatebot/renovate/discussions/26761#discussioncomment-8641152